duplicate_url_discarder/processors/normalize.py

```diff
@@ -0,0 +1,23 @@
+import re
+
+from scrapy.utils.url import parse_url
+
+from .base import UrlProcessorBase
+
+
+class NormalizerProcessor(UrlProcessorBase):
+    def validate_args(self) -> None:
+        if self.args:
+            raise TypeError(f"normalizeUrl doesn't accept args, got: {self.args}")
+
+    def process(self, input_url: str) -> str:
+        """Normalizees the input URL by removing the following:
+
+        * 'www.*' prefixes
+        * trailing slashes
```

Comment:
Maybe we could accept args to allow toggling these.

Reply:
It was my initial approach as well but it would seem that having both yields no difference in the websites compared to only having either one. In fact, we could even turn this on for all websites. Though some quick sanity checks must be made here first.